### PR TITLE
Add Card details

### PR DESCRIPTION
**Pull request by TicTacToyah**

### CODE REVIEW

File: cypress/integration/main.spec.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  describe('section', () => {
+    it('has a card', () => {
+      cy.get('section')
+        .find('h3')
+        .should('have.length', 3)
+      cy.get('section')
+        .find('h2')
+        .should('have.length', 3)
+      cy.get('img').should('have.length', 12)
```
Comment (FSchubert92) [inline cypress/integration/main.spec.js:19]: why lenght 12?

File: src/Card.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const CardBody = styled.section`
+  background: #f2f2f2;
+  box-shadow: 0px 15px 18px rgba(201, 201, 201, 0.5);
+  margin: 15px 0;
+`
+const CardImage = styled.img`
+  width: 100%;
```
Comment (FSchubert92) [inline src/Card.js:11]: what about height?

File: src/Card.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+`
+const CardInformation = styled.section`
+  padding: 5px 10px;
+`
+const CardCategory = styled.h3`
+  font-family: Helvetica;
+  font-style: normal;
```
Comment (FSchubert92) [inline src/Card.js]: you don't need to specify `normal`

File: src/Card.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  display: inline-block;
+  height: 20px;
+  width: 16px;
+`
+const NoseIcon = styled.img`
+  display: inline-block;
+  height: 20px;
+  width: 16px;
+`
+
```
Comment (FSchubert92) [inline src/Card.js:57]: here is a blank line. You might want to add one after every decleration or leave it here as well. I'd suggest to do add one

File: src/Card.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+const EyeIcon = styled.img`
+  display: inline-block;
+  height: 20px;
+  width: 16px;
+`
+
+export default function Card({ category, title, location, smell, optic }) {
+  return (
+    <CardBody>
+      <CardImage src={require('./images/Braune_Bananen_verwerten.jpg')} />
```
Comment (FSchubert92) [inline src/Card.js:67]: it's better to import pictures into the component, otherwise webpack will have some trouble

File: src/Card.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+      <CardInformation>
+        <CardCategory>{category}</CardCategory>
+        <CardHeadline>{title}</CardHeadline>
+        <CardList>
+          <CardListItem>
+            <LocationIcon src={require('./images/location-pin.svg')} alt="" />
```
Comment (FSchubert92) [inline src/Card.js]: especially svg files can be imported like a component

**Review by FSchubert92 — CHANGES_REQUESTED** [review]

File: src/Card.js (hunk 4 above)
Comment (TicTacToyah) [reply src/Card.js:57]: Done

File: cypress/integration/main.spec.js (hunk 1 above)
Comment (TicTacToyah) [reply cypress/integration/main.spec.js:19]: Jeder Karte hat 4 img und auf der Seite sind 3 Karten, daher findet er 12 img

**Pull request merged**